ci-operator/step-registry/cucushift/installer/check/ibmcloud/destroy/cucushift-installer-check-ibmcloud-destroy-commands.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    export region
+    "${IBMCLOUD_CLI}" config --check-version=false
+    echo "Try to login..."
+    "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+ibmcloud_login
+
+CLUSTER_FILTER="${NAMESPACE}-${UNIQUE_HASH}"
+hasError=0
+
+resource_group=$(${IBMCLOUD_CLI} resource groups | awk '(NR>3) {print $1}' | grep ${CLUSTER_FILTER})
+
+echo "Resource group: $resource_group"
+
+if [ ! -z ${resource_group} ]; then
+    "${IBMCLOUD_CLI}" target -g ${resource_group}
+
+    echo "DEBUG" "Checking the remaining resources and the last_operation of the resources in ${resource_group}..."
+    count=$(eval "${IBMCLOUD_CLI}" resource service-instances --type all -g ${resource_group} --output JSON | jq '.[]|.name' | wc -l)
+    if [ "${count}" -gt 0 ]; then
+        echo "ERROR: has remain resource ..."
+        "${IBMCLOUD_CLI}" resource service-instances --type all -g ${resource_group} --output JSON | jq '.[]|.name+" | "+.resource_id+" | "+.crn + " | " +(.last_operation|tostring)'
+        hasError=1
+    fi
+fi
+
+echo "DEBUG" "Checking the remaining dns resource-records ..."
+echo "IBMCLOUD_DNS_INSTANCE_NAME: $IBMCLOUD_DNS_INSTANCE_NAME, BASE_DOMAIN: $BASE_DOMAIN"
+if [ ! -z ${IBMCLOUD_DNS_INSTANCE_NAME} ] && [ ! -z ${BASE_DOMAIN} ]; then
+  cmd="${IBMCLOUD_CLI} dns zones -i ${IBMCLOUD_DNS_INSTANCE_NAME} -o json | jq -r --arg n ${BASE_DOMAIN} '.[] | select(.name==\$n) | .id'"
+  dns_zone_id=$(eval "${cmd}")
+  cmd="${IBMCLOUD_CLI} dns resource-records ${dns_zone_id} -i ${IBMCLOUD_DNS_INSTANCE_NAME} | grep -w ${CLUSTER_FILTER}"
+  count=$(eval "${cmd} -c")
+  if [ "${count}" -gt 0 ]; then
+    echo "ERROR: remaining dns resource-records..."
+    run_command "${cmd}"
+    hasError=1
+  fi
+fi
+
+echo "DEBUG" "Checking the remaining cis dns-records ..."
+ibmcloud_cis_instance_name=$(cat "${CLUSTER_PROFILE_DIR}/ibmcloud-cis")
+if [ ! -z ${BASE_DOMAIN} ]; then
+  cmd="${IBMCLOUD_CLI} cis domains -i ${ibmcloud_cis_instance_name} -o json | jq -r --arg n ${BASE_DOMAIN} '.[] | select(.name==\$n) | .id'"
+  domain_id=$(eval "${cmd}")
+  cmd="${IBMCLOUD_CLI} cis dns-records ${domain_id} -i ${ibmcloud_cis_instance_name} | grep -w ${CLUSTER_FILTER}"
```

Comment:
In the [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/48246/rehearse-48246-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-ibmcloud/1753986720640012288):
```
DEBUG Checking the remaining cis dns-records ...
FAILED
Incorrect Usage. 'DNS_DOMAIN_ID' is required.
```
when there is no cis domain exists for `private-ibmcloud.qe.devcluster.openshift.com`